ovos_plugin_manager/ocp.py

```diff
@@ -73,8 +73,8 @@ def extract_stream(self, uri, video=True):
         return meta or {"uri": uri}
 
 
-if __name__ == "__main__":
-    s = StreamHandler()
-    print(s.supported_seis)
-    # ['rss', 'bandcamp', 'youtube', 'ydl', 'youtube.channel.live',
-    # 'pytube', 'invidious', 'm3u', 'pls']
\ No newline at end of file
+# if __name__ == "__main__":
```

Comment:
What’s going on here?

Reply:
Old test code; I commented out when troubleshooting extra OCP init but it wasn't related. I can replace this if wanted